README.md

```diff
@@ -62,13 +62,36 @@ git clone --recursive https://github.com/clara-genomics/AtacWorks.git
     ```
 ### 3. Tests
 
-Run unit tests:
+Run unit tests to verify that installation was successful
 
     ```
     python -m pytest tests/
     ```
 
+## Workflow
+AtacWorks trains a deep neural network to learn a mapping between noisy (low coverage/low cell count/low quality) ATAC-seq data and matching clean (high coverage, high cell count, and/or high quality) ATAC-seq data. Both noisy and clean data should be from the same cell type or tissue. Once this mapping is learned, the trained model can be applied to improve other noisy ATAC-Seq datasets.
+
+### 1. Training an AtacWorks model
```

Comment:
Should we link to the html documentation instead of the notebooks?

Reply:
These links don't point to the notebooks, they are pointing to md files (which point to notebooks).

To clarify, if the Readme is read through the webpage, it automatically opens the html page (because sphinx compiles and cross-references it). If the Readme is read through the github repo, then it opens the md files.